hub/util/from_tfds.py

```diff
@@ -0,0 +1,43 @@
+import tensorflow  # type: ignore
+import tensorflow_datasets as tfds  # type: ignore
+from tqdm import tqdm  # type: ignore
```

Comment:
Is this listed in the requirements anywhere?

Reply:
Yes in the plugins file.

Reply:
I don't see it in plugins.txt on the branch, can you please double check?